internal/pkg/agent/application/upgrade/step_mark.go

```diff
@@ -160,16 +161,21 @@ func markUpgradeProvider(updateActiveCommit updateActiveCommitFunc, writeFile wr
 			Details:           upgradeDetails,
 		}
 
-		if rollbackWindow > disableRollbackWindow && agent.parsedVersion != nil && !agent.parsedVersion.Less(*Version_9_2_0_SNAPSHOT) {
+		if agent.parsedVersion != nil && !agent.parsedVersion.Less(*Version_9_2_0_SNAPSHOT) {
```

Comment:
I think we can leave this line as-is for this PR?  I know it will change in the follow up PR.  It would also clear the linter warning for `const disableRollbackWindow` above.

Reply:
I am not sure I understand the reason here. The linter error will be taken care of in the follow-up PR, so I wouldn't bother changing it at this point.
The reason why this has been removed from here is to concentrate the logic for available rollbacks in upgrade.go rather than letting `step_mark.go` decide the contents of the upgrade marker.
The reason why this is only done halfway here is because I had to split the original PR commits at some point.